### ISSUE TITLE
Invalid shape for attention bias when serving Bloom

### ISSUE THREAD

**Issue by LinxinS97** [vllm-project/vllm#925]
This error appears when serving BloomChat `sambanovasystems/BLOOMChat`
```
...
File "/root/miniconda3/envs/LLM/lib/python3.10/site-packages/xformers/ops/fmha/common.py", line 120, in validate_inputs
  raise ValueError(
ValueError: Invalid shape for attention bias: torch.Size([14, 10, 10]) (expected (1, 14, 10, 10))
query.shape: torch.Size([1, 10, 14, 128])
key.shape  : torch.Size([1, 10, 14, 128])
value.shape: torch.Size([1, 10, 14, 128])
```
It seems that the shape of attention bias different from the expected in xFormer, which in BloomChat loses the first dimension.

**Comment by LinxinS97** [comment]
Problem solved.
Problem:
The extra dim comes from the `PagedAttentionWithALiBi`, when calling the `xops.memory_efficient_attention_forward`, one dim is added to pass the value check, but bias is not.

Solution:
In `PagedAttentionWithALiBi.set_attn_bias`, change the `[:, :prompt_len, :prompt_len]` to `[None, :, :prompt_len, :prompt_len]` when using `torch.empty` to initialize the bias.

**Comment by WoosukKwon** [comment]
Hi @LinxinS97, which version of vLLM are you using? I thought this problem was fixed in v0.1.4.

**Comment by LinxinS97** [comment]
> Hi @LinxinS97, which version of vLLM are you using? I thought this problem was fixed in v0.1.4.

My version is 0.1.3. Thank you for mentioning this!
